cmd/kyma/install/cmd.go

```diff
@@ -738,8 +738,14 @@ func (cmd *command) printSummary() error {
 		return err
 	}
 
+	var consoleURL string
 	vs, err := cmd.K8s.Istio().NetworkingV1alpha3().VirtualServices("kyma-system").Get("core-console", metav1.GetOptions{})
-	if err != nil {
+	switch {
+	case apiErrors.IsNotFound(err):
+		consoleURL = "not installed"
+	case vs != nil && vs.Spec != nil && len(vs.Spec.Hosts) > 0:
+		consoleURL = fmt.Sprintf("https://%s", vs.Spec.Hosts[0])
+	default:
```

Comment:
This logic looks sound but if the call is correct (err is nil), but there are no hosts on the spec (for whatever reason), you would be returning a nil error and it would be impossible to know what happened and very hard to debug.
Also, it would not work if both `vs` and `err` are not nil. Usually it is not the case, but we can not assume.

I would suggest the following logic to cover those 2 corner cases:
```go
switch {
	case apiErrors.IsNotFound(err):
		consoleURL = "not installed"
	case err != nil:
		return err
	case vs != nil && vs.Spec != nil && len(vs.Spec.Hosts) > 0:
                consoleURL = fmt.Sprintf("https://%s", vs.Spec.Hosts[0])
       default:
                return errors.New("Console host could not be obtained.")
}
```